src/components/ValuePicker/ValueSelectorModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Modal from '@components/Modal';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type {ValuePickerItem, ValueSelectorModalProps} from './types';
+
+function ValueSelectorModal({items = [], selectedItem, label = '', isVisible, onClose, onItemSelected, shouldShowTooltips}: ValueSelectorModalProps) {
+    const styles = useThemeStyles();
+    const [sectionsData, setSectionsData] = useState<ValuePickerItem[]>([]);
+
+    useEffect(() => {
+        const itemsData = items.map((item, index) => ({
+            value: item?.value,
+            alternateText: item?.description,
+            keyForList: item.value ?? '',
+            text: item?.label ?? '',
+            isSelected: item === selectedItem,
+            sectionIndex: 0,
+            index,
```

Comment:
Please make sure these extra params don't affect anything.

Reply:
Removed - not needed anymore